docs/rst/installation/linux/amldashboard.rst

```diff
@@ -0,0 +1,75 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_linux:
+
+#############
+AML Dashboard
+#############
+
+To get started with *AML Dashboard*, follow these steps.
+
+Clone the repository using the following command:
+
+.. code-block:: bash
+
+    git clone https://github.com/eProsima-Private/AML-Dashboard.git
+
+Requirements
+------------
+
+If |amlip| is not already installed in your system, follow the installation instructions provided `here <https://aml-ip.readthedocs.io/en/latest/rst/installation/linux.html>`__.
+
+Python dependencies
+^^^^^^^^^^^^^^^^^^^
+
+Ensure you have **python3.11** installed together with the packages listed in the ``requirements.txt`` file. If not, install them using the following command:
+Ensure you have python3.11 intsalled together with the packages listed in the requirements.txt file. If not, install them using the following command:  
+
+.. code-block:: bash
+    curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -o Miniconda3-latest-Linux-x86_64.sh
+
+    #To activate the conda environment automatically when opening shell. For changes to take effect, close and re-open current shell.
+    bash Miniconda3-latest-Linux-x86_64.sh
+
+    #To activate the conda environment manually each time you open the shell:
+    eval "$(/home/user/miniconda3/bin/conda shell.zsh hook)" 
+
+    conda create --name aml-ip-venv python=3.11
+    conda activate aml-ip-venv
+    cd AML-Dashboard
+    pip3 install -r requirements.txt
```

Comment:
I don't think we should require the use of Conda. If someone wanna have their workspace clean, they can use Docker. 

Additionally, there's some confusion with the `requirements.txt` file. The repository does contain a file with this name, but it isn't the correct one we're referring to. I suggest we list the essential packages directly and provide `pip` installation instructions to maintain consistency with the rest of the documentation.

Reply:
I added a requirements.txt file in the AML-Dashboard Repository.  This is why I added the line 
```
cd AML-Dashboard

```
to make sure that the file used is the correct one. 

Furthermore, the use of Conda is not only to keep the workspace clean, but also to make sure that Python 3.11 is installed, since I have noticed that the AML Engine does not run with Python 3.12, which is the default Python version in Ubuntu 24. Furthermore, a virtual environment is required since pip can not bu used in Ubunut 24.04 outside a virtual environment. I believe we should provide all requirements for anyone who does not want to use Docker.

Reply:
In that case, I believe we should create the virtual environment using `venv` instead of `Conda`. This would keep things simple.

Additionally, we could add a note about the existence of `pipx` for users who prefer that as an alternative for managing Python packages in isolated environments.

Reply:
Done